### ISSUE TITLE
Unable to open context menu using the keyboard

### ISSUE THREAD

**Issue by barklund** [ampproject/amp-wp#3407]
As a user, I want to be able to open the context menu for blocks (both pages and elements) using the keyboard only.

It should be possible to open the context menu when a page or element is selected by pressing Shift+F10 (which [appears to be the reigning standard](https://www.google.com/search?q=accessibility%20context%20menu%20%22shift%20f10%22), though there's no hard specification for this).

### Testing instructions

1. Create a new story
2. Tab until the default empty text block is in focus
3. Hold down Shift and press F10.
4. A context menu should appear near the top left corner of the element

**Comment by spacedmonkey** [comment]
This screencast is from develop on chrome + linux. 
https://youtu.be/8XRz_WWTiUo

**Comment by barklund** [comment]
> This screencast is from develop on chrome + linux.
> https://youtu.be/8XRz_WWTiUo

@spacedmonkey, you tabbed to the rotation handle and opened the context menu. What is the expected behaviour and how did the function perform relative to that?

**Comment by spacedmonkey** [comment]
The expected behaviour is what is seen in google slides. Which is what we have. 
https://youtu.be/C2dVJ-_5l2o

I would say, you tab into the block, once selected, should display when context when is triggered then menu is displayed.

**Comment by csossi** [comment]
Verified in QA:
![image](https://user-images.githubusercontent.com/16945664/66576679-24d9c380-eb46-11e9-8b7d-cfb8c28a602c.png)
